### PR TITLE
#455: Fix for profile images on Twitter no longer loading

### PR DESCRIPTION
**Pull request by mmakarwich10**
- Added another urlReplace on twimg.js to account for the new mapping for the thumbnail and the full-size image.

- Also turned off the pointer-events for the new div that is covering the profile thumbs, and is preventing hoverzoom from accessing the  element.

### CODE REVIEW

File: plugins/twimg.js [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
             'img[src*="twimg.com"]',
             /_(normal|mini)/,
             ''
         );
+        hoverZoom.urlReplace(res,
+            'img[src*="twimg.com"]',
+            /_bigger/,
+            '_400x400'
+        );
+        $('div.css-1dbjc4n.r-1twgtwe.r-sdzlij.r-rs99b7.r-1p0dtai.r-1mi75qu.r-1d2f490.r-u8s1d.r-zchlnj.r-ipm5af').css('pointer-events', 'none');
```
Comment (extesy) [inline plugins/twimg.js]: I'm not sure how stable this change would be. These class names look auto-generated during compile time so they might completely change every time Twitter updates its website. Is there any other way to find the element?

Comment (mmakarwich10) [reply plugins/twimg.js]: Yeah, I thought so too, but rather I thought it changed on login (it doesn't).

But, you could be right, could change on recompile.

I did try to access it with the "nth-child" pseudo element, but apparently JQuery doesn't support that.

I'll look at it again to see if I can find a better way to access it.

Comment (extesy) [reply plugins/twimg.js]: You don't have to find exact element with a single selector. Fetching multiple candidates and filtering through them using some custom javascript would be fine too.

Comment (mmakarwich10) [reply plugins/twimg.js]: True.

Apparently I'm just crazy, I was wrong about JQuery not supporting the "nth-child" pseudo element. I may have been thinking about another project I'm working on that has an older version of JQuery.

Anyway, fixed the selector. Let me know what you think.

Comment (extesy) [reply plugins/twimg.js]: Looks good to me, thank you for the contribution!

Comment (mmakarwich10) [reply plugins/twimg.js]: No prob! I love this extension and I'm happy to help out!

**Pull request merged**